funding/manager.go

```diff
@@ -4532,6 +4532,12 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 		commitFeePerKw = f.cfg.MaxAnchorsCommitFeeRate
 	}
 
+	// Ensure that the commitment feerate is above the minimum relay fee.
```

Comment:
Actually if the `MaxAnchorsCommitFeeRate` is set dynamically, then we won't need to check this value. As at that point, it'll use the min relay fee here instead of the fee rate returned above (that also factors in `feefilter`).

Reply:
@Roasbeef  What about non Anchor transactions?

Reply:
For non anchor, we use the existing logic that'll do more frequent `update_fee` changes.